libcaf_core/test/unit.cpp

```diff
@@ -28,17 +28,23 @@
 
 using namespace caf;
 
-using unit_res_atom = atom_constant<atom("unitRes")>;
-using void_res_atom = atom_constant<atom("voidRes")>;
-using unit_raw_atom = atom_constant<atom("unitRaw")>;
-using void_raw_atom = atom_constant<atom("voidRaw")>;
+using unit_res_atom   = atom_constant<atom("unitRes")>;
+using void_res_atom   = atom_constant<atom("voidRes")>;
+using unit_raw_atom   = atom_constant<atom("unitRaw")>;
+using void_raw_atom   = atom_constant<atom("voidRaw")>;
+using typed_unit_atom = atom_constant<atom("typedUnit")>;
```

Comment:
While aligning the equal signs certainly looks a bit nicer, it's an unnecessary change at the end of the day that only clutters the Git history. Just for future reference.